Dockerfile

```diff
@@ -1,11 +1,11 @@
 # Build stage
-FROM golang:1.21@sha256:549dd88a1a53715f177b41ab5fee25f7a376a6bb5322ac7abe263480d9554021 AS builder
+FROM golang:1.22 AS builder
 COPY . /build
 WORKDIR /build
 RUN CGO_ENABLED=0 go build ./act.go
 
 # Main stage
-FROM python:3.11.5-slim-bullseye@sha256:9f35f3a6420693c209c11bba63dcf103d88e47ebe0b205336b5168c122967edf
+FROM python:3.12.2-slim-bullseye
```

Comment:
Are we sure we want to go to `3.12` now?

Also, is there a `3.12-slim-bullseye` tag that we can use instead of the `3.12.2-slim-bullseye`, which is probably more specific than we want.  The same question applies to `Dockerfile.goreleaser`.